integration-tests/gradle-plugin-test/multi-platform/build.gradle.kts

```diff
@@ -0,0 +1,37 @@
+plugins {
+    kotlin("multiplatform")
+    id("io.realm.kotlin")
+}
+
+kotlin {
+    jvm()
+
+    val hostOs = System.getProperty("os.name")
+    val isMingwX64 = hostOs.startsWith("Windows")
+    val nativeTarget = when {
```

Comment:
I guess this is fine for this project, but this means it is kinda hidden which platform we are testing on. Not sure if it matters though?

Reply:
Yes. I didn't really know what was the best. I actually like that it is just given from the host platform what you are testing on, but maybe it is "too perpendicular" (🤪) to our other testing patterns 🤔 ... I think we could just rework it later if we have a better approach ... Maybe when GHA is in place